assets/js/editor/editor.js

```diff
@@ -30,7 +30,10 @@ class ConnectToBlocksEditorSupport {
 	 */
 	didBlockEditorLoad() {
 		const transformer = new ClassicBlockTransformer();
-		transformer.execute();
+
+		setTimeout(function () {
+			transformer.execute();
```

Comment:
IMO, it's safer to use an interval instead of timeout. We don't know if blocks are always loaded after 100ms.

```suggestion
		const blocksLoadedInterval = setInterval(function () {
			if (document.getElementById('post-title-0')) {
				transformer.execute();
				clearInterval(blocksLoadedInterval);
			}
		}, 100);
```

Reply:
@dsawardekar are you ok with that ^ recommendation?

Reply:
@dinhtungdu I haven't been able to test this yet. But we need to account for cases where an empty post is loaded for example. There may be a better solution here based on Ty's findings above. I want to investigate Gutenberg's bootstrap lifecycle. Planning to get to this later in the week. cc @jeffpaul